jung-algorithms/src/main/java/edu/uci/ics/jung/layout/util/RadiusNetworkNodeAccessor.java

```diff
@@ -30,24 +29,24 @@
 public class RadiusNetworkNodeAccessor<N> implements NetworkNodeAccessor<N> {
 
   private static final Logger log = LoggerFactory.getLogger(RadiusNetworkNodeAccessor.class);
-  protected final Graph<N> graph;
+
```

Comment:
should def not have a Graph in its state, since all the methods pass in a LayoutModel